packages/office-addin-lint/src/lint.ts

```diff
@@ -5,20 +5,18 @@ import * as path from "path";
 
 const esLintPath = require.resolve("eslint");
 const prettierPath = require.resolve("prettier");
-const config = require.resolve("office-addin-lint");
-const configDir = path.parse(config).dir;
 const esLintDir = path.parse(esLintPath).dir;
 const esLintFilePath = path.resolve(esLintDir, "\..\\bin/eslint.js");
 const prettierFilePath = path.resolve(prettierPath, "\..\\bin-prettier.js");
-const esLintConfigPath = path.resolve(configDir, "\..\\src/.eslintrc.json");
+const esLintConfigPath = path.resolve(__dirname, "..\\src\\.eslintrc.json");
```

Comment:
don't put the .eslintrc.json under src. Perhaps it could be in a config subdir instead.

if "config" and "src" are both dirs from the root, I would think this should use "../config/.eslintrc.json" to find the file. 

Remember the .ts will be compiled into the "lib" folder which is where the code will actually run from.